src/lib/pk_pad/raw_hash/raw_hash.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+* (C) 2023 Jack Lloyd
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/internal/raw_hash.h>
+#include <botan/exceptn.h>
+
+namespace Botan {
+
+void RawHashFunction::add_data(const uint8_t input[], size_t length)
+   {
+   m_bits += std::make_pair(input, length);
+   }
+
+void RawHashFunction::final_result(uint8_t out[])
+   {
+   if(m_output_length > 0 && m_bits.size() != m_output_length)
+      {
+      m_bits.resize(0);
+      throw Invalid_Argument("EMSA_Raw was configured to use a " +
```

Comment:
I guess this is a copy and paste error and `EMSA_Raw` should properly be `RawHashFunction`

Reply:
Ah thank you, yes